code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -73,6 +73,9 @@
 			if ((exclusive_roles.len > 0) && !(M.mind.assigned_role.title in exclusive_roles)) // Is the rule exclusive to their job?
 				trimmed_list.Remove(M)
 				continue
+			if(M.mind.pilot_ref) // are they in the vdom?
```

Comment:
```suggestion
			if(!isnull(M.mind.pilot_ref)) // are they in the vdom?
```

Reply:
this is a lil bit heavy of a nit tho, and I disagree because 
a: it becomes needlessly verbose
b: doesn't make it more readable (!isnull itself i think is cursed)
c: the rest of the file uses this, and has plenty more nits i'd like to make, unless you're suggesting I should change them to match

Reply:
change them to match. isnull is more readable in all scenarios except where the var is a boolean

Reply:
how do you determine readability? consider that there are already instances of the bang operator used to see if something doesnt exist, and this would just be reversing them... with more letters